chess/4-database/database.md

```diff
@@ -114,19 +114,70 @@ As part of your unit test deliverable you need to meet the following requirement
 
 For this phase the TAs will grade the quality of your project's source code. The rubric used to evaluate code quality can be found here: [Rubric](../code-quality-rubric.md)
 
+### Common Submission Issues
+
+Since you're adding code that acts with an external program in Phase 4, the likelihood of having auto-grader test
+output different from your local output is higher than previous phases.
+
+If you encounter this, here are some common things to try (expand the collapsible section).
+
+<details>
+
+<summary>Tips for resolving AutoGrader submission issues when local tests pass</summary>
+
+> [!IMPORTANT]
+> This guide assumes that your tests **are passing locally**.\
+> If your tests are not passing on your own computer,\
+> then you should definitely not assume they will pass on the AutoGrader.
+
+#### Database and Table Creation
+
+It's possible your server doesn't create the database and tables correctly when starting up.
+
+Try the following steps to reproduce the problem on your local machine:
+1. Drop your database/schema using an external tool (MySQL shell or workbench).
+1. Rerun the tests locally.
+1. If they fail, double-check the code for creating the database and tables.
+
+#### Hardcoded Database Name
+
+It's fun to assume that your database will always be named "chess", but the AutoGrader will break that assumption!
+
+The auto-grader inserts a new `db.properties` file for grading with most of the values different from those in your file.
+
+Run through the following verifications in your code:
+- Check for any place you may have hardcoded any values from `db.properties`.
+- Check each SQL statement, including where you create tables, for the database name.
+   - For example, use `INSERT INTO table` instead of `INSERT INTO database.table`.
+- The `getConnection` method inside `DatabaseManager` already sets up the connection to use your database,
+  so you shouldn't need to specify the database name if you are using that method to obtain your connections.
+
+#### CasE SEnSiTiVitY
+
+> [!CAUTION]
```

Comment:
This seems redundant with the sentence below. Is it necessary to repeat this here? If it was removed would it make this section more readable?

Reply:
I think the **CAUTION** alert helps identify the significance of the issue, and I agree that the wording could be cleaned up to reduce duplication between the two text bodies.